themes/bootstrap3/templates/search/history-table.phtml

```diff
@@ -60,9 +60,9 @@
               <?=$this->transEsc("history_delete_link")?>
             </a>
           <?php else: ?>
-            <a href="<?=$this->url('myresearch-savesearch')?>?save=<?=urlencode($info->getSearchId())?>&amp;mode=history">
-              <?=$this->icon('search-save') ?>
-              <?=$this->transEsc("history_save_link")?>
+            <a class="icon-link" href="<?=$this->url('myresearch-savesearch')?>?save=<?=urlencode($info->getSearchId())?>&amp;mode=history">
```

Comment:
Looks like you fixed the Save icon here, but not the Delete icon above; see:

![image](https://user-images.githubusercontent.com/309069/197257583-02d16e8f-fcb7-4b89-8598-88c56f229058.png)